docs-chef-io/content/workstation/getting_started.md

```diff
@@ -14,290 +14,295 @@ aliases = ["/workstation_setup.html", "/chefdk_setup.html", "/workstation.html",
+  ```
 
-Use [the chef generate repo]({{< relref "ctl_chef.md#chef-generate-repo" >}}) to
-create the Chef repository. For example, to create a repository called
-`chef-repo`:
+  On macOS systems with Zsh as the default shell, If you use Zsh as your command shell, your configuration file is called `.zshrc`. The command for adding the Chef Ruby path to your `.zshrc` file is:
 
-``` bash
-chef generate repo chef-repo
-```
+  ``` bash
+  echo 'export PATH="/opt/chef-workstation/embedded/bin:$PATH"' >> ~/.zshrc && source ~/.zshrc
+  ```
 
-### Install a Code Editor
-
-A good visual code editor is not a requirement for working with Chef
-Infra, but a good code editor can save you time. A code editor should
-support the following: themes, plugins, snippets, syntax Ruby code
-coloring/highlighting, multiple cursors, a tree view of the entire
-folder/repository you are working with, and a Git integration.
-
-These are a few common editors:
+{{< /foundation_tabs_panel >}}
+{{< foundation_tabs_panel panel-id="ruby-env-windows-panel" >}}
+{{< warning >}}
 
-- [Visual Studio Code (free/open source)](http://code.visualstudio.com)
-- [GitHub Atom - (free/open source)](http://atom.io)
+  On Microsoft Windows, `C:/opscode/Chef Workstation/bin` must be before `C:/opscode/Chef Workstation/embedded/bin` in the `PATH`.
 
-Chef Infra support in editors:
+{{< /warning >}}
+{{< /foundation_tabs_panel >}}
 
-- [VSCode Chef Infra Extension](https://marketplace.visualstudio.com/items?itemName=chef-software.Chef)
-- [Chef on Atom](https://atom.io/packages/language-chef)
+{{< /foundation_tabs_panels >}}
 
-### Starter Kit
+## Setup Your Chef Repo
 
-If you have access to Chef Infra Server through Automate or Chef Manage,
-you can download the starter kit. The starter kit will create the
-necessary configuration files: the `.chef` directory, `config.rb`,
-`ORGANIZATION-validator.pem`, and `USER.pem`. Simply download the
-starter kit and then move it to the desired location on your Chef
-Workstation machine.
+{{% chef_repo_description %}}
 
-## Configure the Chef Repository
+If you're setting up Chef for the very first time **in your organization**, then you will need a Chef repository for saving your cookbooks and other work.
 
-### With WebUI
+Use the [chef generate repo]({{< relref "ctl_chef.md#chef-generate-repo" >}}) command to create your Chef repository. For example, to create a repository called
+`chef-repo`:
 
-Use the following steps to manually set up the chef-repo and to use the
-Chef management console to get the `.pem` and `config.rb` files.
+``` bash
+chef generate repo chef-repo
+```
 
-#### Get Config Files
+## Your Chef Workstation Installation
 
-For a Chef Workstation installation that will interact with the Chef
-Infra Server (including the hosted Chef Infra Server), log on and
-download the following files:
+Chef Workstation installs two different things onto your computer:
```

Comment:
We probably need to reword this. Chef Workstation doesn't actually create the .chef dir. That happens the first time the Workstation App is launched. That's important as a CLI user will not get the directory.